Makefile

```diff
@@ -103,13 +110,25 @@ generate-crds: ## Generate CRDs and Go types using kubebuilder
 	go run sigs.k8s.io/controller-tools/cmd/controller-gen crd object paths=./apis/... output:crd:artifacts:config=config/crd/bases
 	kubectl kustomize config/crd >deploy/crds.yaml
 
+.PHONY: install-crds
+install-crds: ## Install CRDs
+	kubectl kustomize config/crd | kubectl apply -f -
+
+.PHONY: install-gateway-crds
+install-gateway-crds: ## Install Gateway API CRDs
+	$(SELF_DIR)tests/scripts/install-gateway.sh $(GW_API_VERSION) $(ENABLE_EXPERIMENTAL)
+
+.PHONY: uninstall-gateway-crds
+uninstall-gateway-crds: ## Uninstall Gateway API CRDs
+	$(SELF_DIR)tests/scripts/uninstall-gateway.sh $(GW_API_VERSION) $(ENABLE_EXPERIMENTAL)
+
 .PHONY: generate-manifests
 generate-manifests: ## Generate manifests using Helm.
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) $(HELM_TEMPLATE_EXTRA_ARGS_FOR_ALL_MANIFESTS_FILE) -n nginx-gateway | cat $(strip $(MANIFEST_DIR))/namespace.yaml - > $(strip $(MANIFEST_DIR))/nginx-gateway.yaml
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) $(HELM_TEMPLATE_EXTRA_ARGS_FOR_ALL_MANIFESTS_FILE) --set nginx.plus=true --set nginx.image.repository=$(NGINX_PLUS_PREFIX) -n nginx-gateway | cat $(strip $(MANIFEST_DIR))/namespace.yaml - > $(strip $(MANIFEST_DIR))/nginx-plus-gateway.yaml
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) $(HELM_TEMPLATE_EXTRA_ARGS_FOR_ALL_MANIFESTS_FILE) --set nginxGateway.gwAPIExperimentalFeatures.enable=true -n nginx-gateway | cat $(strip $(MANIFEST_DIR))/namespace.yaml - > $(strip $(MANIFEST_DIR))/nginx-gateway-experimental.yaml
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) $(HELM_TEMPLATE_EXTRA_ARGS_FOR_ALL_MANIFESTS_FILE) --set nginxGateway.gwAPIExperimentalFeatures.enable=true --set nginx.plus=true --set nginx.image.repository=$(NGINX_PLUS_PREFIX) -n nginx-gateway | cat $(strip $(MANIFEST_DIR))/namespace.yaml - > $(strip $(MANIFEST_DIR))/nginx-plus-gateway-experimental.yaml
-	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) --set metrics.enable=false --set nginxGateway.productTelemetry.enable=false -n nginx-gateway -s templates/deployment.yaml > conformance/provisioner/static-deployment.yaml
+	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) --set metrics.enable=false --set nginxGateway.productTelemetry.enable=false -n nginx-gateway -s templates/deployment.yaml > config/provisioner/static-deployment.yaml
```

Comment:
do we even need to generate this file as part of the manifests?

Reply:
It's embedded into the provisioner to deploy NGF instances in the conformance tests.

Reply:
yeah I mean if we just generate it before building the binary for the conformance tests with `generate-static-deployment`, does it need to be in this list too?

Reply:
Since the go code embeds the file, I wonder if there would be a compilation error if the file doesn't exist in the repo at all times.

Reply:
oh yeah it fails to compile...so since it's a configuration file that we need wether it's used or not, maybe it makes sense to have it in the `config` directory?

Also to go back to your original point, I had to move it out of the tests directory because `embed` doesn't like it when there are multiples `go.mod`s.

Reply:
Maybe we do `config/tests` just to make it clear?

Reply:
At some point this file may just go away when NGF itself becomes a provisioner. (split data/control planes)

Reply:
Moved it to `config/tests`